File: Advanced/PageAndRDL/CustomTileProvider/VB.NET/TestDesignerPro/HelperForm.vb
Public Class HelperForm

	Private Sub rtfHelp_LinkClicked(sender As Object, e As LinkClickedEventArgs) Handles rtfHelp.LinkClicked
		Process.Start(e.LinkText)
	End Sub

	Private Sub HelperForm_FormClosing(sender As Object, e As FormClosingEventArgs) Handles MyBase.FormClosing
		e.Cancel = True
		Hide()
	End Sub

End Class

File: Advanced/PageAndRDL/CustomTileProvider/C#/TestDesignerPro/HelperForm.cs
using System.Windows.Forms;

namespace GrapeCity.ActiveReports.Samples.TestDesignerPro
{
	public partial class HelperForm : Form
	{
		public HelperForm()
		{
			InitializeComponent();
		}

		private void rtfHelp_LinkClicked(object sender, LinkClickedEventArgs e)
		{
			System.Diagnostics.Process.Start(e.LinkText);
		}

		private void HelperForm_FormClosing(object sender, FormClosingEventArgs e)
		{
			e.Cancel = true;
			Hide();
		}

		private void rtfHelp_TextChanged(object sender, System.EventArgs e)
		{

		}
	}
}
